vcl_templates/www.vcl.erb

```diff
@@ -354,7 +375,11 @@ sub vcl_deliver {
   if (req.http.Cookie !~ "ABTest-EducationNavigation" || req.url ~ "[\?\&]ABTest-EducationNavigation=" || req.url ~ "^/education(\/|\?|$)") {
     add resp.http.Set-Cookie = "ABTest-EducationNavigation=" req.http.GOVUK-ABTest-EducationNavigation "; expires=" now + 365d "; path=/";
   }
-
+  <% ['SearchMatchLength'].each do |test_name| %>
+  if (req.http.Set-Cookie !~ "ABTest-<%= test_name %>" || req.url ~ "[\?\&]ABTest-<%= test_name %>=") {
+    add resp.http.Set-Cookie = "ABTest-<%= test_name %>=" req.http.GOVUK-ABTest-<%= test_name %> "; expires=" now + 1d "; path=/";
```

Comment:
365 just seemed excessive. I was also thinking that searches might happen multiple times with different terms each time.